source/stylesheets/_layout.scss

```diff
@@ -65,11 +64,17 @@
   .awards {
     @include span-columns(3);
     margin-bottom: $footer-height;
+    margin-top: golden-ratio($gutter, -1);
 
     .award {
       @include span-columns(1 of 3);
       @include omega(3n);
-      margin-top: golden-ratio($gutter, -1);
+      margin-bottom: golden-ratio($gutter, -1);
```

Comment:
But what I mean is a variable with a name like `distance-to-top` or whatever the expression means in this context.